neomodel/match.py

```diff
@@ -340,7 +340,10 @@ def _parse_q_filters(self, ident, q):
         target = []
         for child in q.children:
             if isinstance(child, QBase):
-                target.append(self._parse_q_filters(ident, child))
+                q_childs = self._parse_q_filters(ident, child)
```

Comment:
not only minor comment. This block of code - for place the "(" and ")" in correct places. In both sides of OR's conditions